model/src/main/java/de/danoeh/antennapod/model/feed/FeedMedia.java

```diff
@@ -371,7 +371,7 @@ public boolean localFileAvailable() {
     public long getItemId() {
         return itemID;
     }
-
+    
```

Comment:
Unrelated change, please revert

Reply:
@ByteHamster i have this like it was in my code. I just firstly remove this line by mistake and than added it back. Also function isRefreshed which you said is not used, this function is used in ChapterFragment.loadMediaInfo().

Reply:
> i have this like it was in my code

Not entirely. You have space characters that were not there originally.

> Also function isRefreshed which you said is not used, this function is used in ChapterFragment.loadMediaInfo().

Ah, right. Still, I don't understand why this needs to be stored in PlaybackController. Shouldn't this simply be a parameter for `loadMediaInfo`?